homeassistant/components/samsungtv/__init__.py

```diff
@@ -140,7 +145,7 @@ async def _async_create_bridge_with_updated_data(hass, entry):
         updated_data[CONF_PORT] = port
         updated_data[CONF_METHOD] = method
 
-    bridge = _async_get_device_bridge({**entry.data, **updated_data})
+    bridge: SamsungTVBridge = _async_get_device_bridge({**entry.data, **updated_data})
```

Comment:
This is a private method, we should solve the return type of that method instead of overriding the typing here.

```suggestion
    bridge = _async_get_device_bridge({**entry.data, **updated_data})
```